frontend/src/bundles/studio/components/video-menu/components/menu/menu.tsx

```diff
@@ -45,6 +45,7 @@ const Menu: React.FC<Properties> = ({
                 color: 'white',
                 borderTopRightRadius: 'xl',
                 borderBottomRightRadius: 'xl',
+                zIndex: 1000,
```

Comment:
Reduce this zIndex to 100 or 90, since the overlay component has 101 and it should be above everything.